app/controllers/seeds_controller.rb

```diff
@@ -49,7 +49,7 @@ def new
     @seed = Seed.new
 
     # using find_by_id here because it returns nil, unlike find
-    @crop = Crop.find_by_id(params[:crop_id]) || Crop.new
+    @crop = Crop.find_by(id: params[:crop_id]) || Crop.new
```

Comment:
Code Climate is pointing out that alternate_names_controller.rb and ....something else... have practically this same chunk of code in `new()` and I'm left wondering, given existence of `load_and_authorize_resource` and `load_resource` in cancan, do we need to find these by id manually? 

Ok, I'm also wondering why we'd be creating a seed for a crop that doesn't exist.